CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Increase `trivy` memory limits to 1G.
```

Comment:
```suggestion
- Increase `trivy` scan Job memory limits to 1G.
```
Just for clarity. Our actual `trivy` pod memory limit is controlled in the `trivy` [app](https://github.com/giantswarm/trivy-app/blob/master/helm/trivy-app/charts/trivy/values.yaml)